.gitignore

```diff
@@ -1,3 +1,4 @@
 dist/
 node_modules/
 publish/
+.vscode/*
```

Comment:
No change needed, but you might be interested in using a [global gitignore](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer)

Reply:
Ohh that's indeed a cool feature! Added it already :D

Reply:
So can I merge ?

Reply:
I've just left a couple of comments, once those are resolved we can merge.